libstorage/MemoryTable2.cpp

```diff
@@ -214,7 +225,8 @@ int MemoryTable2::insert(
     }
     catch (std::exception& e)
     {
-        STORAGE_LOG(ERROR) << LOG_BADGE("MemoryTable2")
+        // impossible, so exit
+        STORAGE_LOG(FATAL) << LOG_BADGE("MemoryTable2")
```

Comment:
why not use prepare_exit here ?

Reply:
This is pure memory operation, should never failed, if then must be  core